_topic_map.yml

```diff
@@ -359,6 +359,8 @@ Topics:
     File: persistent-storage-aws
   - Name: Persistent storage using Azure Disk
     File: persistent-storage-azure
+  - Name: Persistent storage using Azure File
```

Comment:
This name is consistent with the other assemblies, but procedure modules should start with gerunds.

Reply:
I'm going to leave it as is for now. At some point we need to go through and clean up this content, and we can update the names at this point.